packages/bootstrap/scss/chart-wizard/_variables.scss

```diff
@@ -0,0 +1,29 @@
+// Chart wizard
+
+/// The color of the aria around the chart type icon.
```

Comment:
There is a typo here -- should be area instead of aria ( check all area related variable descriptions )